governance/polls/Ratification Poll - Real-World Finance Core Unit MKR Compensation (RWF-001) - November 8, 2021.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Real-World Finance Core Unit MKR Compensation, RWF-001 (MIP40c3-SP38) - November 8, 2021
+summary: MIP40c3-SP38 adds the MKR Compensation budget for Core Unit RWF-001: Real World Finance.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp38-real-world-finance-core-unit-mkr-compensation-rwf-001/10447
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00
+end_date: 2021-11-22T16:00:00
+---
+# Ratification Poll for Real-World Finance Core Unit MKR Compensation, RWF-001 (MIP40c3-SP38) - November 8, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 8 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP38 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+This proposal contains the MKR Compensation budget for the Core Unit RWF-001: Real-World Finance. This proposal is based on Alternative MKR Compensation Guidelines 6 by @Aes. It provides an average of 80 MKR per FTE per year over 3 years with a 1 year cliff.
```

Comment:
On polls we _italicize_ quoted summaries. Also, I think I'd preserve the link on "[Alternative MKR Compensation Guidelines](https://forum.makerdao.com/t/mip56-alternative-mkr-compensation-guidelines/9230)." And seems like an extraneous "6" has crept in?